sdk/src/client.rs

```diff
@@ -72,6 +72,15 @@ pub trait SyncClient {
         commitment_config: CommitmentConfig,
     ) -> Result<(Hash, FeeCalculator)>;
 
+    /// Get recent fee rate governor
+    fn get_recent_fee_rate_governor(&self) -> Result<FeeRateGovernor>;
```

Comment:
Should we return the matching `blockhash` here?

Do we want a query-by-blockhash interface as well?

Reply:
Seems like it'd be nice if `FeeRateGovernor` was agnostic to an actual fee.  It's just the thing that's used to create a `FeeCalculator`, and if so then `FeeRateGovernor` is not bound to any given blockhash

Reply:
That was my impression as well.

Only user of `FeeRateGovernor` outside a validator is bench-tps, which uses  `max_lamports_per_signature` to estimate the starting account balances.